demo/scripts/controls/sidePane/editorOptions/Plugins.tsx

```diff
@@ -53,6 +55,26 @@ export default class Plugins extends React.Component<PluginsProps, {}> {
                             (state, value) => (state.forcePreserveRatio = value)
                         )
                     )}
+                    {this.renderPluginItem(
+                        'imageEdit',
+                        'Image Edit Plugin',
+                        this.renderCheckBox(
+                            'Size Adaptive Image Handles',
+                            this.sizeAdaptiveImageHandles,
+                            this.props.state.sizeAdaptiveImageHandles,
+                            (state, value) => (state.sizeAdaptiveImageHandles = value)
+                        )
+                    )}
+                    {this.renderPluginItem(
```

Comment:
this causes 3 ImageResizePlugins:
![image](https://user-images.githubusercontent.com/23065085/164553485-9c317c81-c577-4d1a-b699-072cbbf8215a.png)